core/frontend/src/components/health/HealthTrayMenu.vue

```diff
@@ -143,12 +149,20 @@ export default Vue.extend({
     },
   },
   methods: {
-    heartbeat_age(): number {
-      const last_date = mavlink_store_get(mavlink, 'HEARTBEAT.timestamp') as Date
+    heartbeat_age(): number | undefined {
+      const heartbeat = mavlink_store_get(mavlink, 'HEARTBEAT') as MavlinkMessage
+      const last_date = heartbeat.timestamp
       if (last_date === undefined) {
-        return 5000
+        this.last_date_diff = 5000
+        return this.last_date_diff
+      }
+
+      if (heartbeat?.messageData.header.system_id !== 1 && heartbeat?.messageData.header.component_id !== 1) {
```

Comment:
Shouldn't it be `if x or y`? As I understand a vehicle has both system id and component id equal to 1, right?

Reply:
Yep

Reply:
good catch, fixed!